src/main/java/uk/gov/justice/digital/delius/controller/secure/CustodyResource.java

```diff
@@ -83,5 +84,19 @@ public IDs updateOffenderNomsNumber(final @PathVariable String crn,
         return offenderIdentifierService.updateNomsNumber(crn, updateOffenderNomsNumber);
     }
 
+    @RequestMapping(value = "offenders/nomsNumber/{originalNomsNumber}/nomsNumber", method = RequestMethod.PUT, consumes = "application/json")
+    @ApiResponses(value = {
+            @ApiResponse(code = 400, message = "The new noms number is not present"),
+            @ApiResponse(code = 401, message = "Request is missing Authorization header (no JWT)"),
+            @ApiResponse(code = 403, message = "Missing required role for this operation"),
+            @ApiResponse(code = 404, message = "The requested offender was not found")
+    })
+    @ApiOperation(value = "Updates the offender record(s) with the new NOMS number in UpdateOffenderNomsNumber replacing the existing number.",
+            notes = "In the very rare circumstances more than one offender is found with matching noms number, all will be updated and their identifiers returned.")
+    public List<IDs> replaceOffenderNomsNumber(final @PathVariable String originalNomsNumber,
```

Comment:
Decided not to wrap this response, since there really should only be one offender returned, unfortunately Delius still has a handful of offenders that share NOMS number - they have never been able to apply the DB constraint, so theoretically this can return more than one offender